generators/app/templates/typings.json

```diff
@@ -2,6 +2,9 @@
   "globalDependencies": {
     "jasmine": "github:DefinitelyTyped/DefinitelyTyped/jasmine/jasmine.d.ts#dc9dabe74a5be62613b17a3605309783a12ff28a",
 <% if (framework === 'angular1') { -%>
```

Comment:
It was not necessary because the only type used is imported cf https://github.com/FountainJS/generator-fountain-angular2/pull/54/commits/4e15248b94626559c6ad3cd01dca747612ed4b6b#diff-c1d641f63a98c6fd346b5a7804525c3aR17